src/status_im/chats_list/screen.cljs

```diff
@@ -69,16 +69,19 @@
   [chats [:get :chats]]
   [view st/chats-container
    [toolbar-view]
-   [list-view {:dataSource      (to-datasource chats)
-               :renderRow       (fn [[id :as row] _ _]
-                                  (list-item ^{:key id} [chat-list-item row]))
-               :renderFooter    #(list-item [chat-shadow-item])
-               :renderSeparator #(list-item
-                                   (when (< %2 (- (count chats) 1))
-                                     ^{:key (str "separator-" %2)}
-                                     [view st/chat-separator-wrapper
-                                      [view st/chat-separator-item]]))
-               :style           st/list-container}]
+   (let [sorted-chats (sort-by (comp :timestamp :last-message last)
+                               >
+                               chats)]
```

Comment:
I think it is better to move these methods to a separate handler (like `:get-sorted-chats`) instead of sorting everything here.